great_expectations/expectations/core/expect_column_values_to_be_json_parseable.py

```diff
@@ -88,9 +89,25 @@ class ExpectColumnValuesToBeJsonParseable(ColumnMapExpectation):
     }
     args_keys = ("column",)
 
+    @public_api
     def validate_configuration(
         self, configuration: Optional[ExpectationConfiguration] = None
     ) -> None:
+        """Validates the configuration of an Expectation.
+
+        The configuration will also be validated using each of the `validate_configuration` methods in its Expectation
```

Comment:
can you please describe what is required for the configuration to be valid?

Reply:
@donaldheppner no logic has been implemented, so nothing is required at the Expectation-level. I guess if we want we can look though all of the super-class logic and update every Expectation with that?

Reply:
The user is looking at the API docs specifically because they are not looking at the code. I thought it would be most helpful that they know what is needed for config to be valid